composer.json

```diff
@@ -5,6 +5,7 @@
     "keywords": ["DBAL", "ORM", "Database", "Persistence"],
     "homepage": "http://www.doctrine-project.org",
     "license": "MIT",
+    "minimum-stability": "dev",
```

Comment:
I suppose this is ok to avoid a bunch of `composer config` calls in `.travis.yml` - let's see if we have any adverse side effects.